torch/nn/functional.py

```diff
@@ -436,7 +436,7 @@ def fractional_max_pool2d_with_indices(
     _random_samples: Optional[Tensor] = None
 ) -> Tuple[Tensor, Tensor]:
     r"""
-    fractional_max_pool2d(input, kernel_size, output_size=None, output_ratio=None, return_indices=False, _random_samples=None)
+    fractional_max_pool2d(input, kernel_size, output_size=None, output_ratio=None, return_indices=False, _random_samples=None).
```

Comment:
I get your point @mikaylagawarecki. I guess the lint issues are now fixed, please let me know if I should remove the period or not.
Since the description line should end with a period, what can be the way around in these cases then?

Reply:
I think it is okay to remove the period and ignore the pydocstyle errors for these cases, thanks! :)

Reply:
@mikaylagawarecki Done!
Can you please review again and let me know is the PR fit for a merge or not? Let me know if there is need of any other fixes. :))